forms/src/main/java/org/odk/collect/forms/instances/Instance.java

```diff
@@ -83,7 +83,7 @@ public static class Builder {
         private String geometry;
 
         private Long dbId;
-        private boolean canDeleteBeforeSend;
+        private boolean canDeleteBeforeSend = true;
```

Comment:
Do you mean if a form has an entity? We don't currently support local entity creation/updates for synced instances. Otherwise, every `Instance` should now be deletable.